resources/views/announcement.blade.php

```diff
@@ -0,0 +1,29 @@
+@extends('app')
+
+@section('imageUrl')
+{{$vm->imageUrl}}
+@endsection
+
+@section('content')
+    @foreach($vm->data as $ann)
```

Comment:
Now, access the $amaAnnouncement from above directly instead of doing a foreach here.